src/PresentationalComponents/DownloadReport/DownloadReport.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import { DownloadIcon } from '@patternfly/react-icons';
+import './DownloadReport.scss';
+
+export const DownloadReport = () => {
+    return (
+        <Button variant="link" icon={ <DownloadIcon /> }>
+            Report
```

Comment:
lets be sure this value is in Messages.js and there are translations generated for it